op-dispute-mon/mon/extract/extractor_test.go

```diff
@@ -189,10 +191,15 @@ func TestExtractor_Extract(t *testing.T) {
 		require.Equal(t, 1, failed)
 		require.Len(t, enriched, 2)
 		require.Equal(t, 4, enricher.calls)
-		require.Equal(t, enriched[0].Status, gameTypes.GameStatusInProgress)    // Uses cached value from game A
-		require.Equal(t, enriched[1].Status, gameTypes.GameStatusChallengerWon) // Updates game B
-		require.Equal(t, firstUpdateTime, enriched[0].LastUpdateTime)
-		require.Equal(t, secondUpdateTime, enriched[1].LastUpdateTime)
+		// The returned games are not in a fixed order, create a map to look up the game we need to assert
+		actual := make(map[common.Address]*monTypes.EnrichedGameData)
+		for _, data := range enriched {
+			actual[data.Proxy] = data
+		}
+		require.Equal(t, actual[gameA].Status, gameTypes.GameStatusInProgress)    // Uses cached value from game A
```

Comment:
GameInProgress has a zero value. Could we assert that the game actually exists in the map?
Or better yet, change its value to iota + 1

Reply:
Actually made it a non-zero as well so we are completely sure it used the value from after the last update.